src/ui/components/Header/UserOptions.tsx

```diff
@@ -68,10 +56,10 @@ function UserOptions({ setModal, noBrowserItem }: UserOptionsProps) {
           <Icon filename="docs" className="bg-iconColor" />
           <span>Docs</span>
         </button>
-        <button className="row group" onClick={onChatClick}>
+        <ExternalLink className="row group" href="https://discord.gg/n2dTK6kcRX">
           <Icon filename="help" className="bg-iconColor" />
           <span>Chat with us</span>
-        </button>
+        </ExternalLink>
```

Comment:
This has the same visual style but is now a link rather than a button (because that's more semantically correct).